proto/v1/pagination.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option go_package = "./proto";
+package datacommons.v1;
+
+
+message ImportGroupCursor {
+  // The index of the import group.
+  int32 index = 1;
+  // The page index of the cursor, starts from 0.
+  int32 page = 2;
+  // The position of the next read item in the current page, starts from 0.
+  int32 pos = 3;
```

Comment:
name nit:  s/index/import_group/  s/pos/item/

that way, the field names are names of "things" that are indexed.

Reply:
Nice change.